apps/coscrad-frontend/src/components/resources/vocabulary-lists/vocabulary-list-select.tsx

```diff
@@ -0,0 +1,40 @@
+import { IFormField, IValueAndDisplay } from '@coscrad/api-interfaces';
+import { MenuItem, Select } from '@mui/material';
+import { useState } from 'react';
+
+const NO_SELECTION_PLACEHOLDER = '-';
```

Comment:
We should have a `const` that is a default and an optional parameter in the component's props.